KR2k0035_107.txt

```diff
@@ -1,492 +1,133 @@
 # -*- mode: mandoku-view; -*-
 #+TITLE: 欽定熱河志
-#+DATE: 2015-08-24 22:40:17.433824
+#+DATE: 2015-08-24 22:40:17.414766
 #+PROPERTY: ID KR2k0035
 #+PROPERTY: BASEEDITION WYG    
-#+PROPERTY: JUAN 卷一百八
+#+PROPERTY: JUAN 卷一百七
 <pb:KR2k0035_WYG_107-1a>¶
-　欽定四庫全書¶
-　欽定熱河志卷一百八¶
-　　藝文二¶
-　本朝¶
-　　賦¶
-　　　秋郊大獵賦　　　　　　　　　　梁詩正¶
-　　　惟¶
-　聖作都乃文乃武智勇天鍚逺邁前古道洽化淳覃敷¶
+欽定四庫全書¶
+欽定熱河志卷一百七¶
+　藝文一¶
 <pb:KR2k0035_WYG_107-1b>¶
-　　　下土繼志述事遹追¶
-皇祖𬗟惟¶
-謨烈之垂於鑠蒐苗之舉乗農隙而詰戎兵厯周原而臨¶
-　　　津圃時廵經五十餘年賔服暨四十八部匪徒樂乎¶
-　　　從禽實以資乎講旅爾其寒蟲作語殘蝉咽鳴域¶
-　　　中氣爽塞上秋清開廣野而百里聳叠嶂之千層¶
-　　　露㸃山楓丹黄成幄潦收石澗清冽無聲草淺獸¶
-　　　肥則儦儦而俟俟風高馬健斯龎龎而駉駉撫良¶
+¶
```

Comment:
I add only page anchors for juan 107 as there are 30 missing folios, a bit daunting task to prepare the missing text by my own. For anyone interested, the text can be generated from [Wikisource](https://zh.wikisource.org/zh-hant/%E6%AC%BD%E5%AE%9A%E7%86%B1%E6%B2%B3%E5%BF%97_(%E5%9B%9B%E5%BA%AB%E5%85%A8%E6%9B%B8%E6%9C%AC)/%E5%8D%B7107).

Reply:
Thanks for the headsup.  WIll have a look.